### PR TITLE
Change Kennytv's reason + add My Project + Add MagmaMC for modifing EssentialsX Bytecode&abusing Log4j, and add Spongeforge as a alternative of Bukkit+Forge Hybrid

### PR DESCRIPTION
**Pull request by Outfit8TSB**
Proof:
🔭 I mainly work on Paper, ViaVersion <-- , and Hangar. - from kennytv/readme.md
Screenshot_20240624-104413
You Work on ViaVersion means you working on one of dank project called viaversion.
to that continously let evil to live.
3 of My Projects are using GeyserMC Because i only have is Bedrock Edition. i should've created bukkit plugins instead.
(note: i forgot to add spottedleaf for adding kennytv,indirectly merging one of retrooper's pr. so this pull request is draft until spottedleaf is added.)
if this pr won't merge, add me in this list of shame instead, because,
i mashed 2 server softwares, which is PlazmaMC and LeavesMC, in one server software to boycott International Organization.
i also created pull request that dosen't make any sense to turn into a dictator of server software.(eventhough one of responder how to port pufferfish properly without turning my project into dank project, but it turns out it transformed into dank project)
i am also turned into dictator of server software by creating MultiNet(Minestom) and NogyangSpigot(NMS)

### CODE REVIEW

**Comment by Badbird5907** [comment]
huh wheres my free tacos

File: README.md [hunk 1]
```diff
@@ -19,11 +19,9 @@ A non-exhaustive list of dank projects.
 * [Yatopia](https://github.com/YatopiaMC/Yatopia), [Mirai](https://github.com/etil2jz/Mirai) - *[Ouch](https://github.com/kennytv/Yaptapia)*.
 * Hybrid Servers (Forge/Something else + Bukkit) - Servers that combine
   two incompatible APIs and break them both in the process.
+    * [MagmaMC](https://github.com/MagmaFoundation/Magma) - Brake EssentialsX Code([See The Warning](https://essentialsx.net/downloads.html)) and also [Abused log4j bug](https://pixelmonmod.com/viewtopic.php?f=5&t=29799&p=210147)
```
Comment (Badbird5907) [inline README.md:22]: I need to change the brakes on my bike too

**Review by Badbird5907 — CHANGES_REQUESTED** [review]

**Comment by Outfit8TSB** [comment]
@Badbird5907 sorry, this is pull request not tacohouse

**Comment by Badbird5907** [comment]
But I was promised free tacos >:(

**Comment by Outfit8TSB** [comment]
ugh, badbird, NO TACOS HERE!

**Comment by Outfit8TSB** [comment]
kenny merge this or close it and lock it badbird5907 is annoying me

**Comment by Outfit8TSB** [comment]
ok that's it i am pushing more

**Comment by Outfit8TSB** [comment]
you know what this pull is worthless because of badbird5907 asking me free tacos
add me and him later.

**Comment by Badbird5907** [comment]
womp womp

**Pull request closed without merging**